templates/data/mongodb.html

```diff
@@ -0,0 +1,323 @@
+            alt="",
+            width="397",
+            height="256",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/9faa9b91-aws-logo.png",
+            alt="",
+            width="125",
+            height="256",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule">
+    <div class="col">
+      <div class="p-section--shallow">
+        <h2>A hyper-automated MongoDB with enterprise-grade support</h2>
+      </div>
+    </div>
+    <div class="col">
+      <p>The Charmed MongoDB Operator delivers automated operations management from day 0 to day 2 on the MongoDB document database. Charmed MongoDB is an enhanced, open source and fully-compatible, drop-in replacement for the MongoDB Community Edition with advanced MongoDB enterprise features.</p>
+    </div>
+  </div>
+  <div class="row--50-50">
+    <div class="col">
+      <div class="p-strip is-shallow u-align--center u-vertically-center u-aspect-ratio--3-2 u-container--background">
+        {{
+          image (
+            url="https://assets.ubuntu.com/v1/a8147e27-charmed-opensearch-application-operations.svg",
+            alt="",
+            width="230",
+            height="148",
+            hi_def=True,
+            loading="lazy"
+          ) | safe
+        }}      
+      </div>
+    </div>
+    <div class="col">
+      <hr class="p-rule">
+      <div class="p-section--shallow">
+        <ul class="p-list--divided u-no-padding--bottom">
+          <li class="p-list__item is-ticked">Automated security and configuration best practice</li>
+          <li class="p-list__item is-ticked">Made for physical and virtual machines (VMs)</li>
+          <li class="p-list__item is-ticked">Built-in observability</li>
+          <li class="p-list__item is-ticked">Self-healing</li>
+        </ul>
+      </div>
+      <hr class="p-rule">
+      <p>Get started with Charmed MongoDB documentation:</p>
+      <p>
+        <a href="/engage/run-ai-at-scale">Run Charmed MongoDB on IaaS&nbsp;&rsaquo;</a>
+      </p>
+      <p>
+        <a href="/data/docs/mongodb/k8s">Run Charmed MongoDB on Kubernetes&nbsp;&rsaquo;</a>
+      </p>
+    </div>
+  </div>
+</section>
+
+<section class="p-section--shallow">
+  <div class="row--50-50">
+    <hr class="p-rule">
+    <div class="col">
+      <h2>MongoDB enterprise<br class="u-hide--large"> use-cases</h2>
+    </div>
+    <div class="col">
+      <div class="p-section--shallow">
+        <p class="p-heading--5">Web and mobile applications</p>
```

Comment:
```suggestion
        <h3 class="p-heading--5">Web and mobile applications</h3>
```

I'd say make these `h3` since they are headings, for better accessibility and hierarchy. But maybe it's something UX should look at.